gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-http-proxy/src/main/java/io/gravitee/plugin/endpoint/http/proxy/connector/HttpConnector.java

```diff
@@ -101,7 +101,7 @@ public HttpConnector(final HttpProxyEndpointConnectorConfiguration configuration
 
     @Override
     public Completable connect(final ExecutionContext ctx) {
-        return Completable.defer(() -> {
```

Comment:
Same as above. You have been removing all defer of the chain, are you sure the behavior is still the same here?
Request and Response are both mutable.